interop-tests/README.md

```diff
@@ -10,15 +10,27 @@ can dial/listen for ourselves we can do the following:
 
 1. Start redis (needed by the tests): `docker run --rm -it -p 6379:6379
    redis/redis-stack`.
-2. In one terminal run the dialer: `REDIS_ADDR=localhost:6379 ip="0.0.0.0"
+2. In one terminal run the dialer: `redis_addr=localhost:6379 ip="0.0.0.0"
    transport=quic-v1 security=quic muxer=quic is_dialer="true" cargo run --bin ping`
-3. In another terminal, run the listener: `REDIS_ADDR=localhost:6379
-   ip="0.0.0.0" transport=quic-v1 security=quic muxer=quic is_dialer="false" cargo run --bin ping`
+3. In another terminal, run the listener: `redis_addr=localhost:6379
+   ip="0.0.0.0" transport=quic-v1 security=quic muxer=quic is_dialer="false" cargo run --bin native_ping`
 
 
 To test the interop with other versions do something similar, except replace one
 of these nodes with the other version's interop test.
 
+# Running this test with webtransport dialer in browser
+
+To run the webtransport test from within the browser, you'll need the
+`chromedriver` in your `$PATH`, compatible with your Chrome browser.
+Firefox is not yet supported as it doesn't support all required features yet
+(in v114 there is no support for certhashes).
+
+2.
+   - Build the wasm package: `wasm-pack build --target web`
+   - Run the dialer: `redis_addr=127.0.0.1:6379 ip=0.0.0.0 transport=webtransport
+     is_dialer=true cargo run --bin wasm_ping`
```

Comment:
Formatting is slightly off here.

```suggestion
1. Build the wasm package: `wasm-pack build --target web`
2. Run the dialer: `redis_addr=127.0.0.1:6379 ip=0.0.0.0 transport=webtransport is_dialer=true cargo run --bin wasm_ping`
```

Reply:
to be honest I formatted them like so in purpose :sweat_smile: it was meant to be a 'replacement' of step 2 from the paragraph above

Reply:
this is like `2.a` `2.b` when you see them in compiled markdown, just with `-` instead of a and b

Reply:
Ah now I understand. That wasn't clear to me because they are both underneath their own section.

Reply:
I already applied the change, should I revert it?

Reply:
Well, up to you I guess, both makes it